curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The main purpose of the float property in CSS is:
 
 #### --distractors--
 
-Placeholder distractor 1
+To create rounded corners.
 
 ---
 
-Placeholder distractor 2
+To control opacity.
 
 ---
 
-Placeholder distractor 3
+To center vertically.
 
 #### --answer--
 
-Placeholder answer
+To align elements to either the left or the right of their container.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property is used to set the stack order of an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position`
 
 ---
 
-Placeholder distractor 2
+`bg-green`
 
 ---
 
-Placeholder distractor 3
+`float`
 
 #### --answer--
 
-Placeholder answer
+`z-index`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you apply `position: relative` to an element in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It removes the element from document flow.
 
 ---
 
-Placeholder distractor 2
+It only moves the element relative to the viewport.
 
 ---
 
-Placeholder distractor 3
+It positions the element in a fixed position relative to the viewport.
 
 #### --answer--
 
-Placeholder answer
+It moves an element relative to its default position in the document.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property would you use to fix an element at a certain position on the page so that it does not move when scrolling occurs?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position: absolute`
 
 ---
 
-Placeholder distractor 2
+`position: relative`
 
 ---
 
-Placeholder distractor 3
+`display: block`
 
 #### --answer--
 
-Placeholder answer
+`position: fixed`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If an element has `position: absolute`, what element is it positioned relatively to?
 
 #### --distractors--
 
-Placeholder distractor 1
+The closest ancestor with `position: fixed`.
 
 ---
 
-Placeholder distractor 2
+The closest ancestor with `position: absolute`.
 
 ---
 
-Placeholder distractor 3
+The `<body>` element.
 
 #### --answer--
 
-Placeholder answer
+The closest positioned ancestor with `position: relative`, `absolute` or `fixed`.
```

Comment:
```suggestion
The closest positioned ancestor with `position` set to `relative`, `absolute`, or `fixed`.
```
Either it should be both property and value for all three, or just the property and the three different values.